src/components/AppIndex.vue

```diff
@@ -65,35 +119,43 @@ export default {
 
 <style scoped>
 
-ul {
+ul.keywords {
+    display: flex;
+    flex-wrap: wrap;
```

Comment:
This will make the comment swap around.

Reply:
I’m not sure, which comment you mean? What do you mean by swap around?

Reply:
Sorry, typo. I meant **wrap**.

Reply:
Yes, but that’s good, isn’t it? If the keywords don't fit in a single line, the user is still able to see them as in [this picture](https://github.com/nextcloud/cookbook/pull/375#discussion_r519480331).